### PR TITLE
[HOLD] Parallel Write V2

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
@quinthar @mcnamamj @cead22 @flodnv 

This is ready for review.
Replaces [this PR](https://github.com/Expensify/Bedrock/pull/112), which is now closed.

Auth compatibility PR is here: https://github.com/Expensify/Server-Expensify/pull/1858

### CODE REVIEW

**Comment by quinthar** [comment]
To clarify, are you asking for this to be reviewed right now?  My inbox is
asploding so please ping me on Slack when you need my review.

On Wed, Mar 15, 2017 at 5:55 PM, Tyler Karaszewski <notifications@github.com

**Comment by tylerkaraszewski** [comment]
No, not quite yet. I'll tag you as a reviewer here and ping you on slack when it's ready.

**Comment by tylerkaraszewski** [comment]
@quinthar Ready to review.

File: BedrockServer.h [hunk 1]
```diff
@@ -1,141 +1,156 @@
...
+
+    // Send a reply for a completed command back to the initiating client. If the `originator` of the command is set,
+    // then this is an error, as the command should have been sent back to a peer.
+    void _reply(BedrockCommand&);
+
+    // The following are constants used as methodlines by status command requests.
+    static constexpr auto STATUS_IS_SLAVE          = "GET /status/isSlave HTTP/1.1";
+    static constexpr auto STATUS_HANDLING_COMMANDS = "GET /status/handlingCommands HTTP/1.1";
+    static constexpr auto STATUS_PING              = "Ping";
+    static constexpr auto STATUS_STATUS            = "Status";
```
Comment (quinthar) [inline BedrockServer.h]: OMG you can do this in header files now?  :mindblown:

**Comment by mcnamamj** [comment]
🎉

**Pull request merged**